### ISSUE TITLE
Add PyGames Scripts

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/Awesome_Python_Scripts#32]
This issue is specifically created for those who have basic programming knowledge in python and want to explore on API scripts on python

Add basic and advance Games scripts in 'PyGamesScripts' folder.

Repository : PyGamesScripts
Level : Easy and Medium

Do check if the code you want to contribute is available in this repository
Comment on this issue with the title and details of the script you are going to contribute and what does it do
Start working on that script and before you add make sure it works properly
Add your code to 'PyGamesScripts' repository using the contribution guidelines and template mentioned
Create a Pull Request and be patient
We will review your code and if all works perfectly fine, PR will be merged.
If anything else needed from you, we will get back with the response
Make good PR and clean code.
Any queries, drop me a private message or just mention me over here!. I'll be happy to support and mentor you.
Best of luck everyone ❤️

Happy Contribution!

**Comment by MuditCh57** [comment]
Hello!! I'm Mudit. I'm new in opensource. I've created a game with tkinter module. Can I add that game here? And how can I do it any guidance would be very much helpful for me.

**Comment by prathimacode-hub** [comment]
Yes, you are more welcome to add your game script. Create a new issue to state game you are going to contribute. I shall guide you with the details of how it can be done.  @MuditCh57

**Comment by neelshah2409** [comment]
Hello! I am Neel. I am new in opensource. I had made  game with tkinter and numpy.May I add that here?plz guide me.

**Comment by prathimacode-hub** [comment]
Yes sure, create  a new issue and give a slight description of what the project is about. Once assigned it to you, you can start contributing. Do check out the README and Contributing_Guidelines before you create a PR. Once you create a PR, link the issue number to PR as reference and give a commit message. Any queries, feel free to ask. But raise an issue first. Good to see your initiation to contribute. @neelshah2409

**Comment by AK-Subodh** [comment]
Hello @prathimacode-hub ! I'm Anshika. I  am making a  simple oddly-satisfying animation/game using PyGame with basic input input from player.  Can I contribute it here? I'm new to open source, so please guide me about steps. Thank you!

**Comment by prathimacode-hub** [comment]
Ok, sure. You're free to contribution. Raise a new issue with the program you would like to work on after 12AM. Issues would be assigned on first come, first serve basis. And definitely, I would guide you the process as you go. @AK-Subodh

**Comment by Manasa2811** [comment]
Hello @prathimacode-hub , I am Manasa and I'm a participant of LGM SOC'21. I've been working on a simple card game using basic OOP with python. Can I contribute it here?

**Comment by prathimacode-hub** [comment]
Yeah, but raise a new issue using the template mentioned in README file of the project. Else, you can check the assigned issues to know how the template would look like. @Manasa2811

**Comment by vikasbaghel1001** [comment]
Hello @prathimacode-hub i am recently working on a 2d arcade mario style game using pygame library can i contribute it to the pygame script for my hacktoberfest2021 task
